rfd/0026/README.md

```diff
@@ -1274,38 +1289,36 @@ an appropriate error message when used on shared volumes zones.
 Machines acting as shared volumes' storage zones will have the value `nfsserver`
 for their `smartdc_role` property. To know of this new `smartdc_role` value is used by CloudAPI to prevent users from performing operations on these storage VMs, refer to
 
-#### New internal `required_nfs_volumes` property on VM objects
+#### New `volumes` property on VM objects
 
 A VM object that represents a Docker container mounting a shared volumes will
 store a reference to that volume in a new _indexable_ property named
-`required_nfs_volumes`.
+`volumes`.
 
 This will allow VOLAPI to perform an indexed search on VMAPI to determine what
 uses a given volume when handling operations that depends on a volume's usage,
 e.g a `DeleteVolume` request.
 
-This property is _internal_ and is not exposed to VMAPI and CloudAPI users. It
-is used by VMAPI's `ListVms` endpoint to implement support for its new
-[`mounting_volume` input
-parameter](#new-mounting_volume-parameter-for-the-listvms-endpoint).
+This property will also be exposed both at VMAPI and CloudAPI where it can be
+passed in as part of a VM payload to mount specified volumes on VM creation,
+and also can be listed via the ListVms and ListMachines endpoints.
 
-#### New `mounting_volume` parameter for the `ListVms` endpoint
+#### New `volumes` parameter for the `ListVms` endpoint
 
-While the [`required_nfs_volumes`
-property](#new-internal-required_nfs_volumes-property-on-vm-objects) on VM
-objects is internal, the `ListVms` API endpoint allows users to list _active_
-VMs that mount a given volume.
+The ListVms endpoint will include the [`volumes`
+property](#new-volumes-property-on-vm-objects) in order to view the list of
+_active_ VMs that mount a given volume.
 
 ##### Input
 
-| Param           | Type         | Description                              |
-| --------------- | ------------ | ---------------------------------------- |
-| mounting_volume | String       | A string representing _one_ volume UUID for which to list _active_ VMs that reference it. |
+| Param   | Type         | Description                              |
+| ------- | ------------ | ---------------------------------------- |
+| volumes | String       | A string representing _one_ volume UUID for which to list _active_ VMs that reference it. |
```

Comment:
Sorry. I changed the name here without updating the wording. What we want to do here probably depends a lot on the answer to your previous question. If we want to support listing with multiple volumes higher up the stack, we will also want such support in VMAPI I think.